spec/models/spotlight/resource_spec.rb

```diff
@@ -60,10 +84,21 @@ def to_solr
     end
   end
 
-  it 'reindexs after save' do
-    expect(subject).to receive(:reindex)
-    subject.data_will_change!
-    subject.save!
+  describe '#save_and_index' do
+    before do
+      allow(subject.send(:blacklight_solr)).to receive(:update)
+    end
+
+    it 'saves the object and commit to solr' do
```

Comment:
Thanks; we probably don't need to test if `commit` is triggered here, as it's tested as part of `#reindex`. I'll update the test.